### ISSUE TITLE
Wrong capitalization for preemptionPolicy field

### ISSUE THREAD

**Issue by sftim** [kubernetes/website#31262]
**This is a Bug Report**

**Problem:**
In https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#non-preempting-priority-class, the `preemptionPolicy` field is written, incorrectly, as `PreemptionPolicy` (in some places).

**Proposed Solution:**
Fix the capitalization to camelCase.

/sig scheduling
Relevant as there's a plan to [graduate](https://github.com/kubernetes/kubernetes/pull/107432) the feature.

**Page to Update:**
https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/

**Additional Information:**
/language en
/priority backlog

**Comment by denkensk** [comment]
PR for GA list here:
https://github.com/kubernetes/website/pull/31283

**Comment by sftim** [comment]
(In my view), https://github.com/kubernetes/website/pull/31283 should wait for #31267 to merge.

**Comment by denkensk** [comment]
> (In my view), #31283 should wait for #31267 to merge.

@sftim  Yes, I will add `/hold` in https://github.com/kubernetes/website/pull/31283 to wait.
